staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -20,8 +20,11 @@ import (
 	"fmt"
 	"time"
 
+	"gopkg.in/yaml.v2"
```

Comment:
I think we are trying to switch towards using "sigs.k8s.io/yaml" wherever we can

Reply:
You'll also need to run `hack/update-bazel.sh` to fix the BUILD files